### PR TITLE
Cake 4597 | Updates to GDPR Modal

### PR DESCRIPTION
**Pull request by cschucode**
This PR updates the design of the GDPR modal to use a two-screen flow for rejecting cookies. The first screen includes the 'Accept' and 'Learn More' buttons. Clicking on the 'Learn More' button takes you to the second screen with new copy AND the 'Reject cookies' button.

- Updated styles for two-screen GDPR flow
- Added query param `uselang` for testing
- Updated Tests
- Updated Tracking
- Updated Copy

### CODE REVIEW

File: src/ContentManager.js [hunk 1]
```diff
@@ -1,182 +1,158 @@
...
-            'In particular, we want to draw your attention to cookies which are used to market to you, or provide personalized ads to you. Where we use these cookies (“advertising cookies”), we do so only where you have consented to our use of such cookies by ticking "ACCEPT ADVERTISING COOKIES" below. Note that if you select reject, you may see ads that are less relevant to you and certain features of the site may not work as intended.',
-            'You can change your mind and revisit your consent choices at any time by visiting our privacy policy. For more information about the cookies we use, please see our Privacy Policy. For information about our partners using advertising cookies on our site, please see the Partner List below.',
-        ],
+        headline: 'This Site Uses Cookies',
+        bodyParagraphScreenOne: 'Fandom and our partners use technology such as cookies on our site. To consent to these activities, click / tap "ACCEPT ADVERTISING COOKIES." For additional options, please select "LEARN MORE".',
```
Comment (mklucsarits) [inline src/ContentManager.js]: This is minor, but these two quotes use different formatting -- one has the period inside the quotes and the other has it outside. I would assume these should be the same. I'd check with whoever wrote the copy to see which way it should be for both.

File: src/ContentManager.js [hunk 2]
```diff
@@ -1,182 +1,158 @@
...
-            'Puedes cambiar de opinión y revisar las opciones de consentimiento en cualquier momento visitando nuestra política de privacidad. Para obtener más información acerca de los cookies que utilizamos, por favor revisa nuestra política de privacidad. Para obtener más información sobre nuestros socios utilizando las cookies de publicidad en nuestro sitio, consulta la lista de socios abajo.',
-        ],
-        buttonAccept: 'ACEPTAR LOS COOKIES DE PUBLICIDAD',
-        buttonReject: 'RECHAZAR LOS COOKIES DE PUBLICIDAD',
-        privacyLinkText: 'Política de privacidad FANDOM',
+        bodyParagraphScreenOne: 'Fandom y sus socios utilizan herramientas como las cookies en este sitio. Para aceptar el uso de estas herramientas, haz clic en "ACEPTAR COOKIES PUBLICITARIAS". Para ver otras opciones, por favor selecciona "Más Información".',
```
Comment (mklucsarits) [inline src/ContentManager.js]: This translation has inconsistent capitalization of the button text.

File: src/ContentManager.js [hunk 3]
```diff
@@ -1,182 +1,158 @@
...
-        partnerLink: 'http://fandom.wikia.com/partner-list',
+      headline: '本網站使用Cookie',
+      bodyParagraphScreenOne: 'Fandom和我們的合作夥伴在網站中使用了Cookie等技術。 按「接受廣告Cookie」表示您同意這些活動。 查閱更多訊息，請按「瞭解更多」。',
+      bodyPargraphScreenTwo: 'Fandom以及其合作夥伴在網站上將使用Cookie等技術來定制化內容和廣告，提供社交媒體功能，以及分析網站流量。 我們尤其希望您能夠注意到Cookie將用於為您提供訂制廣告。 如果您同意我們使用這些Cookie（「廣告Cookie」），可以按下面的「接收廣告Cookie」按鈕。 請注意，如果您選擇「拒絕」，您可能會看到與您喜好不太相關的廣告，並且網站的某些功能可能無法按正常工作。 您可以隨時透過訪問我們的隱私政策改變您的想法並重新選擇是否同意這項條款。 有關我們使用Cookie的詳細內容，請參閱我們的隱私政策。 關於我們的合作夥伴使用我們網站上的廣告Cookie訊息，請參閱下面的合作夥伴清單。',
+      buttonAccept: '接受廣告Cookie',
+      buttonLearnMore: '閱讀更多',
+      buttonReject: '拒絕廣告Cookie',
+      privacyLinkText: '隱私政策',
+      // privacyLink: 'http://zh.wikia.com/wiki/%E9%9A%90%E7%A7%81%E6%9D%83%E6%96%B9%E9%92%88?variant=zh-hant',
+      privacyLink: 'http://www.wikia.com/Privacy_Policy',
```
Comment (mklucsarits) [inline src/ContentManager.js]: While we're updating this file, we should probably also update this URL to the new Fandom.com version: https://www.fandom.com/privacy-policy

And the partner list as well: https://www.fandom.com/partner-list

File: src/ContentManager.js (hunk 1 above)
Comment (cschucode) [reply src/ContentManager.js]: Good catch. 👍

File: src/ContentManager.js (hunk 3 above)
Comment (cschucode) [reply src/ContentManager.js]: done. https://github.com/Wikia/tracking-opt-in/pull/60/commits/2f9f2f1740e8fbba923efc37e9432310276b52c2

File: src/ContentManager.js (hunk 2 above)
Comment (cschucode) [reply src/ContentManager.js]: https://github.com/Wikia/tracking-opt-in/pull/60/commits/6e90c4ff7b9bdfc940e74d8e403416de05975251

**Pull request merged**